3.9.1/index.d.ts

```diff
@@ -57,8 +23,23 @@ declare interface Gulp {
    * </ul>
    */
   task: typeof Orchestrator.prototype.add;
-  run: RunMethod;
-  watch: typeof watch;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  run: typeof Orchestrator.prototype.start;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  start: typeof Orchestrator.prototype.start;
+  /**
+   * watches all files in the specified glob(s) for changes and runs the given task(s).
+   * @param globs
+   * @param tasks
+   */
+  watch(globs?: string | Array<string>, tasks?: task | Array<task>): EventEmitter;
+  watch(globs?: string | Array<string>, opt?: WatchOptions, tasks?: task | Array<task>): EventEmitter;
```

Comment:
Reference: https://github.com/gulpjs/vinyl-fs/blob/315e76a248e9d150f228e4ef50b61de15dc55123/package.json#L16 and https://github.com/gulpjs/glob-watcher/blob/195a14d51b734fd2ce762f44ec00de97ecb56da7/index.js#L5